CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 🚨 *Breaking Changes*
 
 🔥 *Features*
+* `orq --version` or `orq -v` will now show the current SDK version.
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏If you use the other suggestion for the flag:
```suggestion
* `orq --version` or `orq -V` will now show the current SDK version.
```